src/Microsoft.ML.Data/DataLoadSave/Text/TextLoader.cs

```diff
@@ -1008,19 +1007,29 @@ private bool HasHeader
         private readonly IHost _host;
         private const string RegistrationName = "TextLoader";
 
-        public TextLoader(IHostEnvironment env, Column[] columns, Action<Arguments> advancedSettings, IMultiStreamSource dataSample = null)
-            : this(env, MakeArgs(columns, advancedSettings), dataSample)
+        /// <summary>
+        /// Loads a text file into an <see cref="IDataView"/>. Supports basic mapping from input columns to IDataView columns.
+        /// </summary>
+        /// <param name="env">The environment to use.</param>
+        /// <param name="columns">Defines a mapping between input columns in the file and IDataView columns.</param>
+        /// <param name="hasHeader">Whether the file has a header.</param>
+        /// <param name="separatorChars">Defines the characters used as separators between data points in a row. By default the tab character is taken as separator.</param>
```

Comment:
> By default the tab character [](start = 112, length = 29)

this statement and `char[] separatorChars = null` a bit weird. 
I know what latter down the line we probably check for null in separators, and use tab as default, but still.

Reply:
Yes it is not ideal, that's why I have an explanation in the documentation above. But if you have a better idea, I would be happy to take it!

---
In reply to: [237683995](https://github.com/dotnet/machinelearning/pull/1784#discussion_r237683995) [](ancestors = 237683995)